README.md

```diff
@@ -125,7 +125,11 @@ Releases and tarballs ready to compile are are made available at [GitHub][github
 
  [github-repo]: https://github.com/balabit/syslog-ng/releases
 
-To compile from source, the usual drill applies (assuming you have
+To generate the configure script and compile from source:
```

Comment:
In case of `tar.gz` the configure is already included, so no need for `./autogen.sh`.
I would suggest to simple keep the:
```
To generate the configure script:
```

Or specify it like:
```
In case of not tarballs you could generate the configure with:
```

Reply:
Perhaps leave the text as-is, and instead of prepending a step, append one: `If you don't have a configure script (because of cloning from git, for example), run ./autogen.sh to generate one.`?